communications/telegram/telegram.go

```diff
@@ -43,9 +44,16 @@ const (
 	talkRoot = "GoCryptoTrader bot"
 )
 
+var (
+	// ErrWaiter is the default timer to wait if an err occurs
+	// before retrying after successfully connecting
+	ErrWaiter = time.Second * 30
```

Comment:
I don't think this needs to be exported

Reply:
This is intentionally done for customisability is engine and for standalone apps